bindata/network/ovn-kubernetes/006-ovs-node.yaml

```diff
@@ -56,12 +57,21 @@ spec:
               exit 0
           }
           trap quit SIGTERM
-          /usr/share/openvswitch/scripts/ovs-ctl start --ovs-user=openvswitch:openvswitch --system-id=random
+          setenforce Permissive
```

Comment:
Don't do this.

Reply:
Can you get/paste the selinux denials that drove you to do this?

Reply:
audit2allow provided: 

`
#============= openvswitch_t ==============
allow openvswitch_t container_var_run_t:file { append lock rename };
allow openvswitch_t container_var_run_t:sock_file { create unlink };
allow openvswitch_t spc_t:dir { getattr search };
allow openvswitch_t spc_t:file { getattr open read };
`

Reply:
OK one general approach you can take that the MCD also does for the ostree side is to basically `systemd-run /bin/sh -c ...`, that way you "escape" the `spc_t` type and go back to the default.

Another approach is to run `chroot /host restorecon /path/to/files`.

Reply:
On the `systemd-run` side, since you just have a shell script here, you can copy it out to the host and use `systemd-run` to run all or part from the host side.